tests/user.test.js

```
@@ -25,7 +25,7 @@ describe('User', () => {
     try {
       await sequelize.sync({ force: true });
     } catch (err) {
-      console.log(err);
+      // Error occurred
```

The `catch` block in beforeAll swallows errors silently. If `sequelize.sync({ force: true })` fails (DB connection, missing models, etc.), the test suite will continue with hidden setup failures. Consider at least logging the error or rethrowing to fail fast, e.g. `catch (err) { console.error(err); throw err; }` so problems are visible during CI/local runs.